pkg/proxy/apis/kubeproxyconfig/v1alpha1/types.go

```diff
@@ -148,14 +148,11 @@ type KubeProxyConfiguration struct {
 	ConfigSyncPeriod metav1.Duration `json:"configSyncPeriod"`
 }
 
-// Currently two modes of proxying are available: 'userspace' (older, stable) or 'iptables'
-// (newer, faster). If blank, use the best-available proxy (currently iptables, but may
-// change in future versions).  If the iptables proxy is selected, regardless of how, but
-// the system's kernel or iptables versions are insufficient, this always falls back to the
-// userspace proxy.
+// Currently, four modes of proxying are available total: 'userspace' (older, stable), 'iptables'
+// (newer, faster), 'ipvs', and 'kernelspace' (Windows only, newer).
+//
+// If blank, use the best-available proxy (currently iptables, but may change in
```

Comment:
This needs to call out the difference between Windows and Linux.  iptables doesn't make sense on Windows.

Reply:
I fixed it now. PTAL.